android/src/main/java/com/doordeck/RNDoordeckSdkModule.java

```diff
@@ -40,6 +40,40 @@ public void updateToken(String authToken) {
         Doordeck.INSTANCE.updateToken(authToken, reactContext);
     }
 
+    @ReactMethod
+    public void unlockWithUUID(String uuid) {
```

Comment:
Might be clearer just to call it `deviceId` since we also have `tileId` which is a UUID.